tests/cupy_tests/lib_tests/test_polynomial.py

```diff
@@ -0,0 +1,116 @@
+import unittest
+
+import pytest
+import numpy
+
+import cupy
+from cupy import testing
+
+
+@testing.gpu
+class TestPoly1d(unittest.TestCase):
```

Comment:
Please add tests for each type of array_like inputs.

Reply:
Please address this comment.

Reply:
I added tests for array and poly1d inputs. What other types of 'array_like' inputs should I test ?

Reply:
Please add tests for CPU to GPU conversion as I commented in https://github.com/cupy/cupy/pull/3466#discussion_r444669936.

Reply:
OK, I thought that `test_poly1d_get1` and `test_poly1d_get2` were enough.